subxt/src/custom_values/custom_values_client.rs

```diff
@@ -48,6 +48,24 @@ impl<T: Config, Client: OfflineClientT<T>> CustomValuesClient<T, Client> {
         Ok(value)
     }
 
+    /// Access the bytes of a  custom value by the address it is registered under.
+    pub fn bytes_at<Address: CustomValueAddress + ?Sized>(
+        &self,
+        address: &Address,
+    ) -> Result<Vec<u8>, Error> {
+        // 1. Validate custom value shape if hash given:
+        self.validate(address)?;
+
+        // 2. Return the underlying bytes:
+        let metadata = self.client.metadata();
+        let custom = metadata.custom();
+        let custom_value = custom
+            .get(address.name())
+            .ok_or_else(|| MetadataError::CustomValueNameNotFound(address.name().to_string()))?;
+
+        Ok(custom_value.bytes().to_vec())
+    }
```

Comment:
Looks good! Shame about needing to `to_vec()`, but I can see it's because the metadata is cloned and not referenced; maybe something we can address elsewhere at some point :)